public/404.html

```diff
@@ -1,67 +1,213 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>The page you were looking for doesn't exist (404)</title>
+  <title>Page Not Found | Classroom</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link href='//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
```

Comment:
Is there a reason you're using 100% inline styles? I've seen this technique used with LB-rendered 503's, but since this was rendered from the web-server, I think we can assume that styles can be pulled down from the server as well. I might be missing something though, let me know!